lib/foreman_openscap/engine.rb

```diff
@@ -104,13 +104,15 @@ class Engine < ::Rails::Engine
           permission :destroy_tailoring_files, { :tailoring_files => [:destroy],
                                                  'api/v2/compliance/tailoring_files' => [:destroy] },
                       :resource_type => 'ForemanOpenscap::TailoringFile'
+          permission :openscap_spool, { :smart_proxies => [:openscap_spool] }
```

Comment:
what is this permission used for if we map the action to view smart proxy permission? https://github.com/theforeman/foreman_openscap/pull/235/files#diff-8998f9b7f683bfc0bcb3fe387b588f44R23

if we want to keep it, shouldn't it have a resource specified?